samples/sample_common/src/vaapi_utils_x11.cpp

```diff
@@ -169,14 +147,11 @@ X11LibVA::X11LibVA(void)
 
 X11LibVA::~X11LibVA(void)
 {
-    if (!msdk_strcmp(PROCESSING_DRIVER_NAME, RENDERING_DRIVER_NAME))
+    //release context
+    if (m_contextID != VA_INVALID_ID)
     {
-        //release context
-        if (m_contextID != VA_INVALID_ID)
-        {
-            m_libva.vaDestroyContext(m_va_dpy_render, m_contextID);
-            m_contextID = VA_INVALID_ID;
-        }
+        m_libva.vaDestroyContext(m_va_dpy_render, m_contextID);
```

Comment:
What is m_va_dpy_render if you did not create it? I believe you need to delete this variable as well and use m_va_dpy everywhere.

Reply:
Done, thank you.